kinetic/distribution.yaml

```diff
@@ -1546,6 +1546,25 @@ repositories:
       url: https://github.com/OTL/cv_camera.git
       version: master
     status: developed
+  darknet_ros:
+    doc:
+      type: git
+      url: https://github.com/leggedrobotics/darknet_ros
+      version: master
+    release:
+      packages:
+      - darknet_ros
+      - darknet_ros_msgs
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/leggedrobotics/darknet_ros-release.git
```

Comment:
This repository doesn't seem to exist?

Reply:
Sorry for the mistake. The repository is now online.

Reply:
The repository does not contain a release. I guess you haven't invoked `bloom` yet.